catalogue/webapp/pages/item.js

```diff
@@ -161,6 +162,14 @@ const IIIFViewer = styled.div.attrs(props => ({
   }
 `;
 
+const IframePdfViewer = styled.iframe`
+  width: 90vw;
+  height: 90vh;
```

Comment:
Not sure, just asking - but should all previews take up the same amount of space?
Feels like standardising could be useful, but maybe we're not at the point?

🚲 🏠

Reply:
e.g. Do we have a `PreviewWindow` and everything is just 100% of that?